skimage/measure/_pnpoly.pyx

```diff
@@ -10,7 +10,7 @@ from .._shared.geometry cimport point_in_polygon, points_in_polygon
 cnp.import_array()
 
 
-def _grid_points_in_poly(shape, verts):
+def _grid_points_in_poly(shape, verts, return_labels=False):
```

Comment:
My preference goes to make this hidden function always return the labels and manage the option in the exposed function `measure.pnpoly.grid_points_in_poly` :wink: